<ui version="4.0" >
 <class>MainForm</class>
 <widget class="QDialog" name="MainForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CA SImulation - AiR</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Board</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Actions</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="frame" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btnClear" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>161</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text" >
          <string>Clear Board</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_3" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>140</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="btnSave" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>160</y>
           <width>161</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text" >
          <string>Save board</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_2" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>40</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioAdd" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>60</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text" >
          <string>Add points</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioInsert" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>80</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text" >
          <string>Insert points</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioRemove" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>100</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text" >
          <string>Remove points</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioDesk" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>120</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text" >
          <string>Desk area</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuit" >
        <property name="text" >
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>682</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel" >
     <x>592</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
